Utilities/DataHelper/src/TrackerHitHelper.cpp

```diff
@@ -0,0 +1,114 @@
+#include "DataHelper/TrackerHitHelper.h"
+#include "Identifier/CEPCConf.h"
+
+#include "TMatrixF.h"
+#include "TMatrixFSym.h"
+#include "CLHEP/Matrix/SymMatrix.h"
+#include "CLHEP/Matrix/Matrix.h"
+#include "CLHEP/Vector/ThreeVector.h"
+#include "CLHEP/Vector/Rotation.h"
+#include <bitset>
+
+std::array<float,6> CEPC::GetCovMatrix(edm4hep::TrackerHit& hit){
+  if(hit.isAvailable()){
+    int type = hit.getType();
+    if(std::bitset<32>(type)[CEPCConf::TrkHitTypeBit::COMPOSITE_SPACEPOINT]){
+      return hit.getCovMatrix();
+    }
+    else if(std::bitset<32>(type)[CEPCConf::TrkHitTypeBit::PLANAR]){
+      std::array<float,6> cov{0.,0.,0.,0.,0.,0.};
+      float thetaU = hit.getCovMatrix(0);
+      float phiU   = hit.getCovMatrix(1);
+      float dU     = hit.getCovMatrix(2);
+      float thetaV = hit.getCovMatrix(3);
+      float phiV   = hit.getCovMatrix(4);
+      float dV     = hit.getCovMatrix(5);
+    
+#ifndef MethodUsedInSpacePointBuilder
```

Comment:
Why you are using macro here? 
Tao

Reply:
@mirguest There are two comparing methods to obtain similar results. I try to choose one through macro in default environment, and leave another as reference. And them anyone can test anyone method in their environment. If macro does not recommend to use, it is also ok to remove macro and leave only one.

Chengdong

Reply:
Hi @fucd , using macro is fine. But if you use `#ifdef`, users need to recompile the code in order to use another case. I think we need to think about it and find a better solution. 
Tao

Reply:
@mirguest I moved Identifier to Detector. And I changed the method select by parameter (has a default value), is it better than macro?